# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1522443638 -7200
#      Fri Mar 30 23:00:38 2018 +0200
# Branch stable
# Node ID 74ece2b45b4f7359e27a8374f47d381d8319bc53
# Parent  27ade14df345cf17019ec96aafe7d86ba03a00fb
Revert unintended changes in cset 1cd3aeda7598

* run-octave.in: backout previous changes

diff --git a/run-octave.in b/run-octave.in
--- a/run-octave.in
+++ b/run-octave.in
@@ -102,17 +102,17 @@ fi
 ## in the $builddir/src directory.
 
 OCTAVE_ARCHLIBDIR="$builddir/src"; export OCTAVE_ARCHLIBDIR
 OCTAVE_BINDIR="$builddir/src"; export OCTAVE_BINDIR
 OCTAVE_DEFAULT_QT_SETTINGS="$builddir/libgui/default-qt-settings"; export OCTAVE_DEFAULT_QT_SETTINGS
 OCTAVE_FONTS_DIR="$top_srcdir/etc/fonts"; export OCTAVE_FONTS_DIR
 OCTAVE_JAVA_DIR="$builddir/scripts/java"; export OCTAVE_JAVA_DIR
 OCTAVE_LOCALE_DIR="$builddir/libgui/languages"; export OCTAVE_LOCALE_DIR
-OCTAVE_QTHELP_FILE="$builddir/doc/interpreter/octave_interpreter.qch"; export OCTAVE_QTHELP_FILE
+OCTAVE_QTHELP_COLLECTION="$builddir/doc/interpreter/octave_interpreter.qhc"; export OCTAVE_QTHELP_COLLECTION
 OCTAVE_SITE_INITFILE="$top_srcdir/scripts/startup/site-rcfile"; export OCTAVE_SITE_INITFILE
 OCTAVE_VERSION_INITFILE="$top_srcdir/scripts/startup/version-rcfile"; export OCTAVE_VERSION_INITFILE
 
 if [ "$ADDRESS_SANITIZER_ENABLED" = yes ]; then
   ASAN_OPTIONS=symbolize=1; export ASAN_OPTIONS
 fi
 
 exec $builddir/libtool --mode=execute $driver \
